perun-openapi/openapi.yml

```diff
@@ -730,6 +731,7 @@ components:
         - entitiesToManage
         - assignedObjects
         - assignableToAttributes
+        - displayName
```

Comment:
will this work if display name is null?

Reply:
works for primaryObject so I suppose it should work for displayName as well?

Reply:
right, missed that